docs/en/get_started.md

```diff
@@ -51,7 +51,7 @@ If mmcv and mmcv-full are both installed, there will be `ModuleNotFoundError`.
 1. Create a conda virtual environment and activate it.
 
     ```shell
-    conda create -n openmmlab python=3.7 -y
+    conda create -n openmmlab python=3.10 -y
```

Comment:
We do not want to update the version of python so quickly because many projects do not fully support python 3.10 yet. We should keep 3.7 for a longer time.

Reply:
It's fair. Python3.9 is also fine and has fully Backward Compatibility as well as the older projects. Even now Python3.9 is the default Python for the last release of Anaconda.
I set that to Python3.10 because Ubuntu 22.04 comes with Python3.10. You're right, moving slowly to save stability is a good reason.

Reply:
Agree, then let's change to use 3.9